### DISCUSSION TITLE
Coqui-ai in Spanish

### DISCUSSION THREAD

**Discussion by juanvecino** [coqui-ai/TTS#2327] [General Q&A]
Is there any way I could use the model but with a Spanish voice?

**Comment by erogol** [comment]
There are Spanish models in the library. You can see them by ```tts --list_models``` on terminal. If you mean YourTTS, it doesn't support Spanish ATM.

**Reply by rmcpantoja** [reply]
> There are Spanish models in the library. You can see them by `tts --list_models` on terminal. If you mean YourTTS, it doesn't support Spanish ATM.

Hi @erogol I can help with Spanish support for YourTTS. I've more than ten hours of voice in total from about 18 Spanish speakers. Is it enough or is more needed? Because it is so I can enlarge those datasets.

**Reply by erogol** [reply]
Sure come on discord, we can discuss it there. It'd be nice to add Spanish to YourTTS.
